integration_tests/run_pyspark_from_build.sh

```diff
@@ -289,6 +289,10 @@ else
             jarOpts+=(--packages "${PYSP_TEST_spark_jars_packages}")
         fi
 
+        if [[ -n "$PYSP_TEST_spark_jars_repositories" ]]; then
+            jarOpts+=(--repositories "${PYSP_TEST_spark_jars_repositories}")
```

Comment:
In the prior PRs having the setting twice as an arg and as a key later processed by spark_init_internal was not idempotent. This is why we started unsetting such environment variables. https://github.com/NVIDIA/spark-rapids/pull/6594/files#diff-146d61aa008bf60d3c970d1d76208d85b87070466a6d6021fa63eb7478037a27R305-R309